src/components/TruncatedText.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { TooltipProps } from 'reactstrap';
+import Tooltip from './Tooltip';
+
+
+interface TruncatedTextProps extends TooltipProps {
+  text: string;
+  targetId: string;
+  recordId: string | number;
+  charLimit: number;
+  tooltip: boolean;
```

Comment:
It might be better to include a `tooltipProps` key here. It seems like the top level props is for the truncated text and it so happens to also include tooltipProps.